source/_integrations/matter.markdown

```diff
@@ -405,7 +407,9 @@ This button only exists in the Home Assistant Companion App. It is not available
 Make sure the requirements listed in the [prerequisites](#prerequisites) are met. This includes meeting the minimum system requirements:
 
 - **Android**:
-  - Minimum version is 8.1. Ideally, update to the latest version.
+  - Minimum version is 8.1.
+    - If issues persist, use Android 12 or later.
+      - More issues have been reported by people using older Android versions.
```

Comment:
Maybe here in the minimum requirements we can make it more compact:

Minimum version according to Google is 8.1 but we recommend 12 or later

Reply:
added a note to the system requirements instead